### PR TITLE
Fix umd build

### PR DESCRIPTION
**Pull request by TrySound**
Fixed version
Ref #650 #637 #649

### CODE REVIEW

**Comment by gpbl** [comment]
Could you run `yarn lint --fix` please? 🙏 Thanks!

**Comment by TrySound** [comment]
Done

**Comment by gpbl** [comment]
Thanks man

**Comment by gpbl** [comment]
Sorry, this still doesn't work. Getting the same error https://github.com/gpbl/react-day-picker/issues/649

We need to add a test for making sure `DateUtils` is being exported correctly.

Anyway, which UMD error are you referring to? How can I replicate it?

**Comment by TrySound** [comment]
Weird, I checked this locally. Everything worked fine.

**Comment by gpbl** [comment]
Maybe I pulled something wrong? Let me try again.

**Comment by TrySound** [comment]
Probably. Here's the problem. Just open console
https://codepen.io/TrySound/pen/NYKQMY

**Comment by TrySound** [comment]
@gpbl Did you tried to pull again?
